Content.Server/Spawners/Components/SpawnPointComponent.cs

```diff
@@ -17,6 +17,11 @@ public sealed class SpawnPointComponent : Component
     public SpawnPointType SpawnType { get; } = SpawnPointType.Unset;
 
     public JobPrototype? Job => string.IsNullOrEmpty(_jobId) ? null : _prototypeManager.Index<JobPrototype>(_jobId);
+
+    public override string ToString()
+    {
+        return $"{_jobId} {SpawnType}";
+    }
```

Comment:
is this used for anything?

Reply:
If you view a list of C# objects in the debugger it will use the ToString method to make debugging nicer.